### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by mohamedSabry0**
In this project, I implemented the CRUD (create, update, delete) methods. All the elements of the user interface are fully functional and the application is completed.

### CODE REVIEW

File: src/modules/task.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Task {
-  constructor(description, completed, index){
+  constructor(description, completed, index) {
```
Comment (victorokoroji) [inline src/modules/task.js:2]: - [ ] When I tried to add a new item, I got this error from the console 👇 

image

from 👇 

image

I  see you were trying to access the description attribute at the call of the addTask() function which has not been created yet hence why it is throwing an error and unable to add a new task. Kindly check it out 👍

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi  @mohamedSabry0 , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- You have written a very clean code ✅
- Your code is modular and DRY. Keep it up ✅
- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] Kindly fix the requested change so that the remaining functionalities of the app be tested properly.

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/task.js (hunk 1 above)
Comment (mohamedSabry0) [reply src/modules/task.js:2]: Thank you for the feedback! I'm not sure why I can't re-create the issue. Nevertheless, I'm going to submit another review request after committing the likely fix.

**Review by mwanawabangona — APPROVED** [review]
Hi @mohamedSabry0 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**